rust/osm2lanes/src/transform.rs

```diff
@@ -378,119 +412,82 @@ fn bicycle(
             ));
         }
     }
-
-    // My brain hurts. How does the above combinatorial explosion play with
```

Comment:
Hard to follow from the JSON -> YAML diff. Did some of these broken or edge cases get implemented better, or was the test added to the ignore-list for now?

Reply:
I could actually delete it without breaking anything, will make a separate PR to demosntrate :)

Reply:
https://github.com/a-b-street/osm2lanes/pull/33